.pre-commit-config.yaml

```
@@ -1,51 +1,56 @@
 exclude: Pipfile\.lock|migrations|\.idea
 
 repos:
-  - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.0.1
-    hooks:
-      - id: trailing-whitespace
-      - id: check-executables-have-shebangs
-      - id: check-json
-      - id: check-case-conflict
-      - id: check-toml
-      - id: check-merge-conflict
-      - id: check-xml
-      - id: check-yaml
-      - id: end-of-file-fixer
-      - id: check-symlinks
-      - id: mixed-line-ending
-      - id: fix-encoding-pragma
-        args:
-          - --remove
-      - id: pretty-format-json
-        args:
-          - --autofix
-      - id: requirements-txt-fixer
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.0.1
+  hooks:
+  - id: trailing-whitespace
+  - id: check-executables-have-shebangs
+  - id: check-json
+  - id: check-case-conflict
+  - id: check-toml
+  - id: check-merge-conflict
+  - id: check-xml
+  - id: check-yaml
+  - id: end-of-file-fixer
+  - id: check-symlinks
+  - id: mixed-line-ending
+  - id: fix-encoding-pragma
+    args:
+    - --remove
+  - id: pretty-format-json
+    args:
+    - --autofix
+  - id: requirements-txt-fixer
 
-  - repo: https://github.com/PyCQA/isort
-    rev: 5.9.1
-    hooks:
-      - id: isort
-        files: \.py$
-        exclude: ^build/.*$|^.tox/.*$|^venv/.*$
-        args:
-          - --lines-after-imports=2
-          - --multi-line=3
-          - --trailing-comma
-          - --force-grid-wrap=0
-          - --use-parentheses
-          - --ensure-newline-before-comments
-          - --line-length=88
+- repo: https://github.com/PyCQA/isort
+  rev: 5.9.1
+  hooks:
+  - id: isort
+    files: \.py$
+    exclude: ^build/.*$|^.tox/.*$|^venv/.*$
+    args:
+    - --lines-after-imports=2
+    - --multi-line=3
+    - --trailing-comma
+    - --force-grid-wrap=0
+    - --use-parentheses
+    - --ensure-newline-before-comments
+    - --line-length=88
 
-  - repo: https://github.com/PyCQA/flake8
-    rev: 3.9.2
-    hooks:
-      - id: flake8
+- repo: https://github.com/PyCQA/flake8
+  rev: 3.9.2
+  hooks:
+  - id: flake8
 
-  - repo: https://github.com/ambv/black
-    rev: 21.6b0
-    hooks:
-      - id: black
-        args:
-          - --safe
+- repo: https://github.com/ambv/black
+  rev: 21.6b0
+  hooks:
+  - id: black
+    args:
+    - --safe
+
+- repo: https://github.com/dhruvkb/ymlfmt
+  rev: 66026fc
+  hooks:
+  - id: ymlfmt
```

An alternative to having to maintain a separate repository for yaml formatting (including updates, etc) we could use the [prettier pre-commit hook](https://prettier.io/docs/en/precommit.html#option-3-pre-commithttpsgithubcompre-commitpre-commit) which already supports yaml.

```suggestion
- repo: https://github.com/pre-commit/mirrors-prettier
  rev: v2.5.1
  hooks:
  - id: prettier
    types_or: [yaml]
```

A bonus is that we already use prettier in WordPress/openverse and WordPress/openverse-frontend.

---

This is great, I wish I knew sooner. I'll look into this.

---

Changed the hook to use Prettier in 58492dc.